src/nimble/NimbleBluetooth.cpp

```diff
@@ -299,6 +306,65 @@ void NimbleBluetooth::sendLog(const uint8_t *logMessage, size_t length)
     logRadioCharacteristic->notify(logMessage, length, true);
 }
 
+// Convert the decimal format of latitude and longitude to NMEA format (degrees and minutes)
+void convertToNMEAFormat(double value, char* buffer, int bufferSize, int isLatitude) {
+    int degrees = (int)value;
+    double minutes = (value - degrees) * 60.0;
+    
+    if (isLatitude) {
+        snprintf(buffer, bufferSize, "%02d%07.4f", degrees, minutes);
+    } else {
+        snprintf(buffer, bufferSize, "%03d%07.4f", degrees, minutes);
+    }
+}
+
+void NimbleBluetooth::Send_GPWPL(uint32_t node, char* name, int32_t latitude_i, int32_t longitude_i)
```

Comment:
Please re-use and possibly improve the code in `/src/gps/NMEAWPL.*` because the same functionality exists on the serial port.